### DISCUSSION TITLE
Which installation type is recommended?

### DISCUSSION THREAD

**Discussion by masterflitzer** [ImageMagick/ImageMagick#7435] [General] [answered]
Hi,

I searched around a bit, but couldn't find an answer to my question.
So which installation type is recommended and are there (dis)advantages?

- I checked the GitHub release page and found an .msixbundle, but no .exe file (https://github.com/ImageMagick/ImageMagick/releases)
- I checked the website and found multiple .exe files, but no .msixbundle (https://imagemagick.org/script/download.php#windows)

From the .exe files there is a clear recommendation, so my question is should i go for `ImageMagick.Q16-HDRI.msixbundle` or `ImageMagick-7.1.1-34-Q16-HDRI-x64-dll.exe`?

Overall i find it a little confusing...

Also if I may ask, why is the download location not unified or every file offered on both locations?

Thank you in advance

**Comment by dlemstra** [comment, marked as answer]
The `.msixbundle` file(s) is used by WinGet. It was more convenient to put this on our releases page where we also publish the AppImage for Linux.

It all depends on what you are using ImageMagick for. When you are using it together with other software you might need to use the `.exe` version of the installer. For other cases you can probably install it using WinGet. We will be going to add more versions there in the future.

**Reply by masterflitzer** [reply]
Thank you for the answer, I understand it better now

However if I may, can you clarify the winget situation more? Because I am a huge fan of package managers so i looked there first, but found that winget returns multiple results when running `winget search imagemagick`, the most relevants are these two:

- `ImageMagick.ImageMagick`: manifest for the .exe
- `ImageMagick.Q16-HDRI`: manifest for the .msixbundle

the main version (with just the name imagemagick), installs the .exe even though you said the .msixbundle is made for winget

the manifests can be found in this repo: https://github.com/microsoft/winget-pkgs/tree/master/manifests/i/ImageMagick

**Reply by dlemstra** [reply]
That `ImageMagick.ImageMagick` manifest was not created by us and I can't just replace it with the `msixbundle` because that might break other people their installation when they upgrade.

**Reply by masterflitzer** [reply]
ok makes sense, thank you very much for the clarification and quick replies, i'll try the .msixbundle and if it doesn't work fall back to .exe
